closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -472,6 +472,12 @@ private static void setProperties(Properties props) {
         propsMapping.put("jdk.tls.legacyAlgorithms", restricts.jdkTlsLegacyAlgorithms);
         propsMapping.put("jdk.certpath.disabledAlgorithms", restricts.jdkCertpathDisabledAlgorithms);
         propsMapping.put("jdk.security.legacyAlgorithm", restricts.jdkSecurityLegacyAlgorithm);
+        String fipsMode = System.getProperty("com.ibm.fips.mode");
+        if (fipsMode == null) {
+            System.setProperty("com.ibm.fips.mode", restricts.jdkFipsMode);
+        } else if (!fipsMode.equals(restricts.jdkFipsMode)) {
+            printStackTraceAndExit("Property com.ibm.fips.mode is incompatible with semeru.customprofile and semeru.fips properties.");
```

Comment:
Other uses of `printStackTraceAndExit()` don't include the trailing period; suggest this shouldn't either.

Reply:
I did search the code base for this and about 1/3 used periods and the rest did not. I removed the period from the message just now in this update.

Reply:
Consistency would have been nice, but it's probably not critical (nor worth dealing with in this pull request).